### ISSUE TITLE
Return route with placeholders for angular ngResource?

### ISSUE THREAD

**Issue by ioleo** [FriendsOfSymfony/FOSJsRoutingBundle#119]
Is it possible to configure JsRouting to return route with placeholders?

It would be useful to have it for [angular ngResource](http://stackoverflow.com/a/13275194/741949).

E.g. route:

```
get_user:
    path:       /api/users/{id}.{_format}
    defaults:   { _controller: AcmeDemoBundle:User:getUser, _format: json }
    options:    { expose: true }
    requirements:
        id:  \d+
```

Would be exposed as:

```
/api/users/:id.json
```

Which could be used by `ngResource` to call GET for:

```
/api/users.json      <-- if id passed to ngResource = null
/api/users/6.json     <-- if id passed to ngResource = 6
```

And also call POST, PATCH, PUT and DELETE for:

```
/api/users/6.json     <-- if id passed to ngResource = 6
```

Ofcourse I can do that already by adding the uri in my script. But what if I change my routing in symfony2 app? I have to edit my script. That sucks. Useing this bundle would be very nice.

So, to summarize:

1) is this already possible? if so, how?
2) if not -> that would be a nice feature
